prometheus-config-reloader/Dockerfile

```diff
@@ -8,8 +8,8 @@ FROM registry.access.redhat.com/ubi9/ubi:9.4
 ARG UPSTREAM_VERSION
 ARG RELEASE
 
-ENV  SUMMARY="UBI based prometheus-config-reloader" \
-    DESCRIPTION="Prometheus configuration reloader"
+ENV  SUMMARY="Prometheus Configuration Reloader" \
+    DESCRIPTION="Prometheus Configuration Reloader"
```

Comment:
Can we have a longer description?

Reply:
We could, however it wasn't the intention of this PR to clean up description labels

Reply:
I get it, but it kinda hurts the eye to see that these are the same 🙈 Can we somehow make sure that this will be fixed eventually?

Reply:
I updated description to be longer and also added `UBI based` to all summary and change `name` to the proper image name. Will push it soon (after tests)